.travis.yml

```diff
@@ -7,10 +7,9 @@ stages:
   - test
 
 php:
-  - '5.6'
-  - '7.0'
```

Comment:
Did travis drop support for 5.6 and 7.0?

Reply:
No, Laravel 5.8 did. PHP 5.6 and PHP 7.0 is EOL and doesn’t receive any updates anymore.

The new void return declarations with Laravel 5.8 will only work with PHP 7.1 and above.